PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -40,7 +40,7 @@
     inputTensorName= cms.string("float_input"),
     outputNames = cms.vstring(["probGOOD", "wpMedium", "wpTight"]),
     batch_eval =cms.bool(True),
-    outputFormulas = cms.vstring(["at(1)", "? at(1) > 0.14 ? 1 : 0", "? at(1) > 0.35 ? 1 : 0"]),
+    outputFormulas = cms.vstring(["at(1)", "? at(1) > 0.08 ? 1 : 0", "? at(1) > 0.20 ? 1 : 0"]),
```

Comment:
Should we preserve the old formula for Run2 reprocessing by adding it back through the `run2_nanoAOD_ANY` modifier?

Reply:
ok cool